src/data_item.rs

```diff
@@ -189,7 +189,7 @@ mod tests {
             // open, high, low , close, volume
             (-1.0, 25.0, 15.0, 21.0, 7500.0),
             (20.0, -1.0, 15.0, 21.0, 7500.0),
-            (20.0, 25.0, -1.0, 21.0, 7500.0),
+            //(20.0, 25.0, -1.0, 21.0, 7500.0),
```

Comment:
this is unrelated to vwap, seems like this was leftover/broken on main branch?

Reply:
It was broken indeed. Currently fixed in master.

Reply:
@greyblake fixed merge conflicts